tests/src/test/fdc3.raiseIntent.ts

```diff
@@ -1,32 +1,150 @@
 import { ResolveError } from "@finos/fdc3";
-
-const ExpectedErrorNotThrownError = "Expected error NoAppsFound not thrown";
+import { assert, expect } from "chai";
 
 export default () =>
-  describe("fdc3.raiseIntent", async () => {
-    it("Method is callable", async () => {
-      const context = {
-        type: "ThisContextDoesNotExist",
-        name: "Name",
-        id: {
-          ticker: "ticker",
-          ISIN: "US0378331005",
-          CUSIP: "037833100",
-          FIGI: "BBG000B9XRY4",
+  describe("fdc3.raiseIntent", () => {
+    before(async () => {
+      await window.fdc3.getOrCreateChannel("fdc3.raiseIntent");
+      await window.fdc3.joinChannel("fdc3.raiseIntent");
+    });
+
+    beforeEach(async () => {
+      await window.fdc3.broadcast({ type: "closeWindow" });
+      await new Promise<void>((resolve) => setTimeout(() => resolve(), 1000)); // Wait until close window event is handled
+    });
+
+    it("Should start app B when raising intent 'sharedTestingIntent1' with context 'testContextY'", async () => {
+      const intentResolution = (await window.fdc3.raiseIntent(
+        "sharedTestingIntent1",
+        {
+          type: "testContextY",
+        }
+      )) as any;
+
+      expect(intentResolution.source).to.eq("IntentAppBId");
```

Comment:
This doesn't actually guarantee the target application was opened. I'd use the same approach as used in the 'open' tests, where the mock broadcasts that it has been opened:

https://github.com/finos/FDC3-conformance-framework/blob/146b281c46e89254c34849e63d6bef5f0e05fb1d/mock-app/index.html#L17

And the tests await this message:

https://github.com/finos/FDC3-conformance-framework/blob/main/tests/src/test/fdc3.open.ts#L29